lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1963,9 +1962,10 @@ definitions:
     minLength: 1
     required:
       - cell_id
-      - device_class
-      - transmit_enabled
```

Comment:
not having `transmit_enabled` required for managed radios doesn't make any sense from an API semantics point of view